CHANGELOG.md

```diff
@@ -4,10 +4,15 @@ All notable changes to this project will (in theory) be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+### Fixed
+- Fixed the `mfa` property in the UserResponse for GET /user
+
 ## [6.0.0]
-### Breaking Changes
+### Added
```

Comment:
Since this does include backwards-breaking changes, let's still include a clear indicator of that somewhere in this release's details.

Reply:
Ok, I thought that was inferred by ### Changed and that ### Breaking Changes doesn't fit the keepachangelog format. I'll make a note in the comments.

Reply:
(Also implied by the major version bump)